ui/app/adapters/named-path.js

```diff
@@ -47,11 +47,12 @@ export default class NamedPathAdapter extends ApplicationAdapter {
   // GET request with list=true as query param
   async query(store, type, query) {
     const url = this.urlForQuery(query, type.modelName);
-    const { paramKey, filterFor } = query;
+    const { paramKey, filterFor, allowed_client_id } = query;
```

Comment:
guessing we're doing a mix here of camelCase and snake_case because of the params defined on the endpoint?

Reply:
Yeah! Exactly - I thought it made it clearer that the first two are frontend specific, whereas the `allowed_client_id` is sent to the backend. Thoughts?